js/options.js

```diff
@@ -58,4 +58,31 @@
             });
         });
     });
+
+    // Apply i18n
+    $(document).ready(function(){
+        // Basic Substitution
+        $('[data-i18n]').each(function(){
+            var $this = $(this);
+            $this.text(i18n($this.data('i18n')));
+        });
+
+        // Text with link to Play Store
+        var $installSignalLinkContent = $('<span>' + i18n('installSignalLink') + '</span>');
+        $installSignalLinkContent.find('a').attr({
+            class: 'link',
+            href: 'https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms',
+            target: '_blank'
+        });
+        $('#installSignalLink').append($installSignalLinkContent);
+
+        // Text with link to Twitter
+        var $installFollowUsContent = $('<span>' + i18n('installFollowUs') + '</span>');
+        $installFollowUsContent.find('a').attr({
+            class: 'link',
+            href: 'https://twitter.com/whispersystems',
+            target: '_blank'
+        });
+        $('#installFollowUs').append($installFollowUsContent);
```

Comment:
We may want to refactor out some of the common code here, perhaps by creating another function / extending `i18n` to allow for attribute replacement, and then generate the html rather than a jQuery object.

Perhaps usage could be something like this?

```javascript
var installFollowUsHtml = i18n('installFollowUs', {
    // any jQuery selector
    'a': {
        class: 'link',
        href: 'https://twitter.com/whispersystems',
        target: '_blank'
    }
});
$('#installFollowUs').html(installFollowUsHtml);
```